### ISSUE TITLE
morpheus: failed/incorrect parses on forms of δείδω/δείδια

### ISSUE THREAD

**Issue by balmas** [alpheios-project/morpheus#7]
δέδιτον (dual) and ἐδεδίειμεν (1st plural pluperfect indicative) as reported in our paradigm table do not produce results for morpheus

**Comment by balmas** [comment]
@monzug can you update this item based upon your most recent findings?

**Comment by monzug** [comment]
accordingly to Smith (pag 184, section 317), our 'Bible' now:
1) δέδιτε paradigm table says this should be 2nd plural indicative but Morpheus reports it as 2nd plural imperative: NO information on this ending, the only two reported are δέδιμεν (1st person plural) and δεδίᾱσιν (3rd person plural) in Smith - Note that we SHOULD change this last ending as in both Smith and Morpheus is  δεδίᾱσι(ν)  but in Mastronarde is δέδιᾱσῐ(ν)
Vanessa added some forms in https://github.com/alpheios-project/inflection-tables/issues/314
δεδίατε is perf ind act 2nd pl
2) No information on δέδιτον (dual) in Smith or from Vanessa's forms
3) δέδιμεν is Plural in Smith, so Morpheus and Vanessa have it as infinite so we go by Smith, right?
4) ἐδεδίειμεν (1st plural pluperfect indicative) in Mastronarde table: No information in Smith on this specific form, we only have the ἐδέδισαν 3rd person PLURAL and ἐδεδίειν 1st person SINGULAR.
in Vanessa's forms (and Morpheus), we have ἐδείδιμεν (plup ind act 1st pl (epic)), but it's epic. Should we add it?
There is also the DUAL ἐδεδίτην  that is missing in Mastronarde  and Morpheus

I did try to play around with accents to see if I was able to find the endings in Morpheus but not much luck...

based on our conversation, the forms that are not in Smith are most likely missing in literature or not so frequent. maybe we should add an asterisk with a footnote????
to sum up: 
I would correct the δεδίᾱσιν (3rd person plural) ending accordingly to Smith and δεδίατε accordingly to Vanessa, then add the DUAL ἐδεδίτην  to the verbpadgm17c table
how about ἐδείδιμεν as it's epic?

**Comment by monzug** [comment]
let-s wait for the 'real' Smyth to arrive...

**Comment by monzug** [comment]
need to review this one, based on Smyth too (not only Smith)

**Comment by monzug** [comment]
see also https://github.com/alpheios-project/alpheios-core/issues/65

1) δέδιτε is  2nd plural indicative also in Smyth -  Morpheus should be updated  --- TO DO
2) δέδιτον (dual) is also reported in Smyth -  Morpheus should be updated ----- TO DO
3) δέδιμεν is 1st person plural indicative in Smyth - Morpheus should be updated  ---- TO DO
Note that the infinitive is δεδιέναι in Morpheus, Smyth and Mastronarde
4) the Pluperfect endings are very different in Smyth from Mastronarde  ----- shall we update/add the endings?
if we update (or add) the endings, Morpheus is missing the duals ἐδέδιτον   and    ἐδεδίτην, the plural  ἐδέδιμεν   and ἐδέδιτε

here is the table from Smyth (section 703 page 200)

Perfect | Pluperfect | Subjunctive
-- | -- | --
δέδοικα or δέδια | ἐδεδοικη or ἐδεδίη | δεδίω (rare)
δέδοικας or δέδιας | ἐδεδοίκης or ἐδεδίης | Optative
δέδοικε or δέδιε | ἐδεδοίκει or ἐδεδίει | δεδιείην (rare)
δέδιτον | ἐδέδιτον | Imperative
δέδιτον | ἐδεδίτην | δέδιθι (poet.)
  |   | Infinitive
δέδιμεν or δεδοίκαμεν | ἐδέδιμεν | δεδιέναι or δεδοικέναι
δέδιτε or δεδοίκατε | ἐδέδιτε
δεδίᾱσι or δεδοίκᾱσι | ἐδέδισαν or ἐδεδοίκεσαν | Participle
  |   | δεδιώς, -υῖα, -ός or
  |   | δεδοικώς, -υῖα, -ός.
